### PR TITLE
#26: Image component with Zoom

### PR DESCRIPTION
**Pull request by BeeMargarida**
| - | - |
| --- | --- |
| Issue | #26 |
| Dependencies | -- |
| Decisions | Image component with support for zoom and pivot point. (Component with zoom on hover will be made in another PR). |
| Animated GIF | Oct-15-2020 12-17-25 |

### CODE REVIEW

**Comment by gcandal** [comment]
Check the prop forwarding and getting rid of the `div` we discussed

**Comment by BeeMargarida** [comment]
@gcandal Regarding the removal of the `div` and setting width and height, it does not work, an overflow happens (scroll bars in page).
Screenshot 2020-10-19 at 15 53 40

File: vue/test/setup.js [hunk 1]
```diff
@@ -1,4 +1,3 @@
-require("@babel/polyfill");
```
Comment (gcandal) [inline vue/test/setup.js]: This is the kind of thing that must go in a separate PR, even though it's a pain to all of us :(

File: package.json [hunk 2]
```diff
@@ -58,7 +58,7 @@
     },
     "dependencies": {
         "loaders.css": "^0.1.2",
-        "ripe-sdk": "^1.12.1",
```
Comment (gcandal) [inline package.json]: Same as https://github.com/ripe-tech/ripe-sdk-components-vue/pull/29/files#r507935804

**Review by gcandal — APPROVED** [review]

**Review by joamag — APPROVED** [review]
